test/k8sT/istio.go

```diff
@@ -29,46 +29,51 @@ import (
 // This tests the Istio integration, following the configuration
 // instructions specified in the Istio Getting Started Guide in
 // Documentation/gettingstarted/istio.rst.
-var _ = SkipContextIf(func() bool {
-	return helpers.SkipQuarantined() && (helpers.GetCurrentK8SEnv() == "1.19" || helpers.GetCurrentK8SEnv() == "1.20")
-}, "K8sIstioTest", func() {
+var _ = Describe("K8sIstioTest", func() {
 
 	var (
 		// istioSystemNamespace is the default namespace into which Istio is
 		// installed.
 		istioSystemNamespace = "istio-system"
 
-		istioVersion = "1.5.9"
+		istioVersion = "1.6.14"
 
 		// Modifiers for pre-release testing, normally empty
-		prerelease     = "" // "-beta.1"
+		prerelease     = "" // "-beta.10"
 		istioctlParams = ""
+
 		// Keeping these here in comments serve multiple purposes:
 		// - remind how to test with prerelease images in future
 		// - cause CI infra to prepull these images so that they do not
 		//   need to be pulled on demand during the test
-		// " --set values.pilot.image=docker.io/cilium/istio_pilot:1.5.9" +
-		// " --set values.proxy.image=docker.io/cilium/istio_proxy:1.5.9" +
-		// " --set values.proxy_init.image=docker.io/cilium/istio_proxy:1.5.9"
+		// " --set values.pilot.image=docker.io/cilium/istio_pilot:1.6.14" +
+		// " --set values.global.proxy.image=docker.io/cilium/istio_proxy:1.6.14" +
+		// " --set values.global.proxy_init.image=docker.io/cilium/istio_proxy:1.6.14" +
+		// " --set values.global.proxy.logLevel=trace"
+		// " --set values.global.logging.level=debug"
+		// " --set values.global.mtls.auto=false"
 		ciliumOptions = map[string]string{
 			// "proxy.sidecarImageRegex": "jrajahalme/istio_proxy",
+			// "kubeProxyReplacement": "disabled",
```

Comment:
This should be enabled?

Reply:
It is left to the whatever the default is for the current integration (e.g., GKE). I left it commented there as a reminder for testing future releases as there may be significant interaction between this setting and policy enforcement with Istio sidecars.